package.json

```diff
@@ -20,7 +20,7 @@
   },
   "scripts": {
     "postinstall": "lerna bootstrap",
-    "build": "lerna run compile",
+    "build": "lerna run compile && rm ./packages/daemon/dist/migrations/*.d.ts",
```

Comment:
Why remove the declaration files?

Reply:
Because `sequelize-cli` will load everything under `migrations`, this will throw an error when it encounters `.d.ts` since the interpreter of `sequelize-cli` is a JS engine.

Reply:
I see